permissions/index.js

```diff
@@ -0,0 +1,23 @@
+const users = require('./users');
+const semesters = require('./semesters');
+const modules = require('./modules');
+const courses = require('./courses');
+const forums = require('./forums');
+const log = require('../helpers/logging');
+
+const handleAccessDenied = (req, res) => {
+  log.error('Access Denied');
+  req.flash('error', 'Access Denied');
+  res.status(403);
+  res.redirect('back');
+};
+
+const ensureAuthorised = funct => (req, res, next) => {
+  if (funct(req)) {
+    next();
+  } else {
+    handleAccessDenied(req, res);
+  }
+};
+
+module.exports = { helpers: { ...users, ...semesters, ...modules, ...courses, ...forums }, ensureAuthorised };
```

Comment:
Maybe should call these `checkers` to prevent confusion with `/premissions/helpers`. Only start to call them `helpers` when adding them into `app.locals`.